src/test/resource/archive1.txt

```diff
@@ -0,0 +1 @@
+Hello world
```

Comment:
I think that this file is not necesary

Reply:
To zip a file this file have to be empty